packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.tsx

```diff
@@ -100,37 +108,46 @@ const RenderForm = ({
         </Text>
       </ModalHeader>
       <ModalContent className={styles.content}>
```

Comment:
From watching the video it looks a bit jarring. What do you think about using `ModalContentPages`? I know there's a different footer too though

Reply:
good call!